app/Models/User.php

```diff
@@ -4,18 +4,20 @@
 
 // use Illuminate\Contracts\Auth\MustVerifyEmail;
 use App\Observers\UserObserver;
-use Filament\AvatarProviders\UiAvatarsProvider;
+use Filament\Auth\MultiFactor\App\Contracts\HasAppAuthentication;
+use Filament\Auth\MultiFactor\App\Contracts\HasAppAuthenticationRecovery;
```

Comment:
Is this only email-based MFA, or can it be configured to use other types? (My preference would be to default to a passkey or app and only fallback to email/SMS.)